### PR TITLE
license: change to BSL v1.1

### PR DESCRIPTION
**Pull request by anikdhabal**
change to BSL v1.1 from GPL V3
/claim #434

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,674 +1,143 @@
...
+Business Source License 1.1
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+-----------------------------------------------------------------------------
+
+Parameters
+
+Licensor:             Arguflow, Inc.
```
Comment (skeptrunedev) [inline LICENSE.txt:12]:
```suggestion
Licensor:             Devflow, Inc.
```

File: LICENSE.txt [hunk 2]
```diff
@@ -1,674 +1,143 @@
...
+Licensed Work:        Arguflow Network Virtualization Engine 1.4.4
```
Comment (skeptrunedev) [inline LICENSE.txt]:
```suggestion
Licensed Work:        Arguflow API, search, and chat
```

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
Please also squash into one commit titled `license: change to BSL v1.1`

For future reference, all commits in our repository are typically tagged with `license:`, `feature:`, `bugfix:` etc. as specced in the issue.

**Comment by skeptrunedev** [comment]
To squash and re-push: 

```
git reset --soft @~2
git commit -m "license: change to BSL v1.1"
git push --force
```

**Comment by PratikMMohite** [comment]
Also their is MariaDB mentioned all over license file.

File: LICENSE.txt [hunk 3]
```diff
@@ -1,674 +1,143 @@
...
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+-----------------------------------------------------------------------------
+
+Parameters
+
+Licensor:             Arguflow, Inc.
+Licensed Work:        Arguflow
```
Comment (skeptrunedev) [inline LICENSE.txt:13]:
```suggestion
Licensed Work:        Arguflow API, search UX, and chat UX
```

**Review by skeptrunedev — CHANGES_REQUESTED** [review]

File: LICENSE.txt [hunk 4]
```diff
@@ -1,674 +1,143 @@
...
+                      Software Foundation
+											https://www.apache.org/licenses/
+
+Alternative Licensing
+
+If you would like to use the Licensed Work in any way that conflicts with
+the stipulations of the Additional Use Grant, contact Arguflow, Inc. to
+obtain an alternative commercial license.
+
+Visit us on the web at: https://www.Arguflow.ai/
```
Comment (skeptrunedev) [inline LICENSE.txt:60]:
```suggestion
Visit us on the web at: https://arguflow.ai/
```

File: LICENSE.txt (hunk 3 above)
Comment (anikdhabal) [reply LICENSE.txt]: Done

File: LICENSE.txt (hunk 1 above)
Comment (anikdhabal) [reply LICENSE.txt]: Done

**Review by skeptrunedev — CHANGES_REQUESTED** [review]

**Comment by skeptrunedev** [comment]
A few small requests for change on this PR: 



> Also their is MariaDB mentioned all over license file.

This is normal as they were the original creators of the BSL license :)

File: LICENSE.txt (hunk 4 above)
Comment (anikdhabal) [reply LICENSE.txt:60]: ok

**Comment by anikdhabal** [comment]
@skeptrunedev all done

**Review by skeptrunedev — APPROVED** [review]
Great work @anikdhabal

**Comment by skeptrunedev** [comment]
> Also their is MariaDB mentioned all over license file.

Make sure to star the repo by the way! :wink:

**Comment by anikdhabal** [comment]
> 😉

Yeah definitely 😉

**Pull request merged**